dashboard/app/models/peer_review.rb

```diff
@@ -208,4 +209,8 @@ def self.get_potential_reviews(script, user)
       level_source_id: PeerReview.where(reviewer: user, script: script).pluck(:level_source_id)
     )
   end
+
+  def clear_data
+    update(data: SYSTEM_DELETED_DATA)
```

Comment:
As there might be value in us knowing the data was cleared manually rather than simply (always) being blank. That said, I've adjusted `SYSTEM_DELETED_DATA` to be the empty string.